### ISSUE TITLE
Tags with diacritical signs don't display completely in the list of tags

### ISSUE THREAD

**Issue by ericguirbal** [xwmx/nb#328]
Here is an example.
```
nb❯ bookmark https://www.delscookingtwist.com/fr/chili-au-quinoa-et-a-la-courge-butternut
 --tags quinoa,végétarien,épicé
Added: [1] 🔖 20240712024012.bookmark.md "Chili au quinoa et à la courge butternut - Del's 
cooking twist (www.delscookingtwist.com)"
 ```
The tags with diacritical signs don't display completely. If the diacritical sign is on the first letter, the tag doesn't show up.

```
nb❯ list --tags
#quinoa
#v
```

However the search command works:

```
nb❯ search -t épicé
[1] 🔖 Chili au quinoa et à la courge butternut - Del's cooking twist (www.dels)
--------------------------------------------------------------------------------
11:#quinoa #végétarien #épicé
```

**Comment by xwmx** [comment]
@ericguirbal I was able to reproduce this in Ubuntu 20.04, but it works as intended in Ubuntu 24.04 and macOS. It's not clear to me yet why it isn't working. It seems like `git grep` is exiting when it gets to a non-ascii character, but `git` is at the latest version on all systems and the encoding is set properly.

What is your OS and OS version?

**Comment by ericguirbal** [comment]
My OS is the latest Debian stable, Debian Bookworm (git version 2.39.2).

**Comment by xwmx** [comment]
This appears to be due to `nb` using `git grep --perl-regexp`, which apparently exits when it matches a non-ascii character. I've updated the `git grep` call to use `--extended-regexp` instead and updated the tests, and the version in the repository appears to be working now. This change will be in the next release.

**Comment by ericguirbal** [comment]
I've already tested it. It works well. Thanks.

**Comment by xwmx** [comment]
This change is now available in versions 7.14.0 and higher. @ericguirbal Thanks for your help with this!
